package.json

```diff
@@ -107,7 +107,7 @@
     "@graphql-codegen/typescript-apollo-client-helpers": "^2.1.10",
     "@graphql-codegen/typescript-operations": "^2.2.4",
     "@graphql-codegen/typescript-react-apollo": "^3.2.5",
-    "@playwright/test": "^1.40.0",
+    "@playwright/test": "^1.47.0",
```

Comment:
We needed an update here to solve this issue on Apple Silicon Mcas https://github.com/microsoft/playwright/issues/29399